beacon_node/network/src/sync/backfill_sync/mod.rs

```diff
@@ -613,7 +614,31 @@ impl<T: BeaconChainTypes> BackFillSync<T> {
             BatchProcessResult::FaultyFailure {
                 imported_blocks,
                 penalty,
+                faulty_component,
             } => {
+                let Some(responsible_peers) = batch.responsible_peers() else {
+                    crit!("Shouldn't happen");
```

Comment:
Should this be an error instead? i.e. do we want this kind of error to trigger pager?

```suggestion
                    error!(
                        batch_id = batch_id.as_usize(),
                        "Responsible peers not found for a failed batch"
                    );
```